test/jasmine/tests/legend_scroll_test.js

```diff
@@ -219,14 +238,10 @@ describe('The legend', function() {
         });
 
         it('should resize when traces added', function(done) {
-            legend = document.getElementsByClassName('legend')[0];
-            bg = document.getElementsByClassName('bg')[0];
-            var origLegendHeight = getLegendHeight();
+            var origLegendHeight = getLegendHeight(gd);
 
             Plotly.addTraces(gd, { x: [1, 2, 3], y: [4, 3, 2], name: 'Test2' }).then(function() {
-                legend = document.getElementsByClassName('legend')[0];
-                bg = document.getElementsByClassName('bg')[0];
```

Comment:
The subplot bg layer now also has class `bg`, which made this query grab the wrong node.